go/cron/pfflow.go

```diff
@@ -0,0 +1,158 @@
+package maint
+
+import (
+	"net/netip"
+	"time"
+)
+
+//easyjson:json
+type PfFlowHeader struct {
+	AgentAddr      netip.Addr `json:"agent_addr"`
+	Timestamp      uint64     `json:"timestamp"`
+	FlowSeq        uint32     `json:"flow_seq"`
+	DomainID       uint32     `json:"domain_id"`
+	SubAgentID     uint32     `json:"sub_agent_id"`
+	SysUptime      uint32     `json:"sys_uptime"`
+	FlowType       uint16     `json:"flow_type"`
+	SampleInterval uint16     `json:"sample_interval"`
+	EngineType     uint8      `json:"engine_type"`
+	EngineId       uint8      `json:"engine_id"`
+}
+
+//easyjson:json
```

Comment:
Remove

Reply:
This is needed for code generation with easyjson